lib/backend/k8s/syncer.go

```diff
@@ -884,12 +885,7 @@ func (syn *kubeSyncer) parseNamespaceEvent(e watch.Event) []model.KVPair {
 	return []model.KVPair{*rules, *tags, *labels}
 }
 
-func (syn *kubeSyncer) parseNodeEvent(e watch.Event) (*model.KVPair, *model.KVPair) {
-	node, ok := e.Object.(*k8sapi.Node)
-	if !ok {
-		log.Panicf("Invalid node event. Type: %s, Object: %+v", e.Type, e.Object)
-	}
-
+func splitNodeEvent(node *k8sapi.Node) (*model.KVPair, *model.KVPair) {
```

Comment:
minor naming nit: I'd just call this `splitNode` since it's not actually taking an `Event` struct as an input, rather a Node.